app/assets/javascripts/homepage/conversation.coffee

```diff
@@ -0,0 +1,97 @@
+conversation = angular.module("Conversation", ["ngRoute", "SharedServices"])
+
+conversation.config(["$routeProvider", ($routeProvider) ->
+    $routeProvider.when("/conversation/:id", {
+        templateUrl: "/assets/conversation.html",
+        controller: "ConversationController"
+    })
+
+]).directive("ngEscape", ->
```

Comment:
Don't prefix a directive with "ng" if it's not from angular. Probably prefix with "my" or something.